src/services/axios/index.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
```

Comment:
this was copied over from 'src/api/index.ts' - this is still valid and may need some clean up later.
currently the various api import axios from various locations ... abstracting this under HttpService gives more consistency/control over the underlying implementation